imap_processing/quality_flags.py

```diff
@@ -0,0 +1,63 @@
+"""Bitwise flagging."""
+
+from enum import IntFlag
+
+
+class L1bQualityFlags(IntFlag):
+    """Quality flags."""
+
+    NONE = 0x0
+
+    BASE_INF = 0x1  # bit 0, Infinite value
+    BASE_MISSING_TELEM = 0x2  # bit 1, Missing telemetry
+    BASE_NEG = 0x4  # bit 2, Negative value
+    BASE_RES1 = 0x8  # bit 3, Reserved 1
```

Comment:
I find specifying the values using hex no clearer than decimal values. `2**n` where `n` is the bit number is clearer IMO.